rust/Cargo.lock

```diff
@@ -733,7 +733,7 @@ dependencies = [
 
 [[package]]
 name = "connlib-android"
-version = "0.1.6"
+version = "1.20231001.0"
```

Comment:
`cargo` will complain if you change it's lockfile like this :D

Reply:
Hmm this was generated from a `cargo check` though. Is it because the version number is too long?

Reply:
Ah, I see. Nevermind it is correct then. I didn't look closely enough and though your `sed`s where targeting all `version = ` lines under `rust`/.